### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by jsoref**
This is a fairly manual process.
I'm mostly walking through the document linearly. In theory it would be possible to fold certain changes together eventually, but....

I have a specific target in mind (a certain item is written w/ confused tense), but it's fairly far off into the future.

I've stopped at `.chapter "Incorporated code" "CHID2"` (which is <2% through the document).

Probably the most important thing is in fact not really a grammar change at all. The section about PGP signing is totally inaccurate. It describes a key which was obsolete years ago and points to a url which resolves to a system that no longer recognizes the base key, it is also apparently no longer identifying the current maintainer. -- Note: I'm not remotely attached to my changes, please feel free to replace them w/ whatever suits the project.

One reason I'm posting now is that this document is fairly large, and completion is the enemy of the good. I can't guarantee I'll finish reviewing this document, so I figure it's worth offering what I have now.

### CODE REVIEW

**Comment by HeikoSchlittermann** [comment]
Hi Josh,
Josh Soref <notifications@github.com> (Di 12 Dez 2017 10:57:10 CET):
> This is a fairly manual process.
> I'm mostly walking through the document linearly. In theory it would be possible to fold certain changes together eventually, but....
> 

thank you for your work and contributions. We'll investigate it after
we've pushed the current release (4.90) out to the public. This (the
release) is expected for the very near future.

-- 
Heiko

File: doc/doc-docbook/spec.xfpt [hunk 1]
```diff
@@ -896,9 +889,7 @@ has been down, and it also maintains per-host retry information.
 .section "Policy control" "SECID11"
 .cindex "policy control" "overview"
 Policy controls are now an important feature of MTAs that are connected to the
-Internet. Perhaps their most important job is to stop MTAs being abused as
-&"open relays"& by misguided individuals who send out vast amounts of
```
Comment (p) [inline doc/doc-docbook/spec.xfpt]: Above two lines should be reinstated I think.

File: doc/doc-docbook/spec.xfpt [hunk 2]
```diff
@@ -1676,7 +1667,7 @@ A C99-capable compiler will be required for the build.
 .cindex "PCRE library"
 Exim no longer has an embedded PCRE library as the vast majority of
 modern systems include PCRE as a system library, although you may need
-to install the PCRE or PCRE development package for your operating
+to install the PCRE package or PCRE development package for your operating
```
Comment (p) [inline doc/doc-docbook/spec.xfpt]: `or the` ?

**Comment by p** [comment]
@HeikoSchlittermann  I was looking to make some documentation additions in SMTP auth, can this PR and #55 be merged first? I can look into rebasing/resolving conflicts.

**Comment by jsoref** [comment]
Sorry, this is a rebase w/ the two things @p noted addressed and some conflicts handled.

**Comment by p** [comment]
Ping, any of the maintainers alive?

**Comment by HeikoSchlittermann** [comment]
Oleg Pudeyev <notifications@github.com> (Do 07 Jun 2018 05:28:58 CEST):
> Ping, any of the maintainers alive?
Pong.
Yes. Busy. But alive.
    Best regards from Dresden/Germany
    Viele Grüße aus Dresden
    Heiko Schlittermann
-- 
 SCHLITTERMANN.de ---------------------------- internet & unix support -
 Heiko Schlittermann, Dipl.-Ing. (TU) - {fon,fax}: +49.351.802998{1,3} -
 gnupg encrypted messages are welcome --------------- key ID: F69376CE -
 ! key id 7CBF764A and 972EAC9F are revoked since 2015-01 ------------ -

**Comment by HeikoSchlittermann** [comment]
Merged into master

**Pull request closed without merging**